packages/bpk-docs/src/helpers/tokens-helper.js

```diff
@@ -69,21 +67,30 @@ export const getTokenValue = (token, platform) => {
   return value || '-';
 };
 
-export const getTokens = (tokens, keys = null) => {
-  const outTokens = {};
-  (keys || Object.keys(tokens)).forEach(key => {
-    outTokens[key] = tokens[key] || null;
-  });
-  return outTokens;
-};
+export const getTokens = (tokens, keys = null) =>
+  _.chain(keys || Object.keys(tokens))
+    .reduce((acc, key) => {
+      if (!tokens[key]) return acc;
+      acc[key] = tokens[key];
+      return acc;
+    }, {})
+    .sortBy(token => {
+      const numericMatch = `${token.value}`.match(/([0-9]*[.]?[0-9]+)/g);
+      if (!numericMatch) {
+        return -1;
```

Comment:
this puts all the non numeric matches close to each other, will they then be sorted by key

Reply:
Web shadows are still in the wrong order here:

![image](https://user-images.githubusercontent.com/1440090/39295840-54291ac6-4937-11e8-8fe0-9c4af899e507.png)

For non -numeric values sorting by key alphabetically isn't correct because we want `xs` to come before `sm`. Perhaps we should look at an alternative to sorting by value? Adding `order` meta data to token source might be the way to go?